helm-charts/falcon-sensor/templates/container_deployment.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.container.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: injector
+  namespace: falcon-system
+  labels:
+    app: injector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: injector
+  template:
+    metadata:
+      labels:
+        app: injector
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1234
```

Comment:
would `RUNASUSER > 10000` work?

ref https://kubesec.io/basics/containers-securitycontext-runasuser/

Also ref https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups

Reply:
I'll look into this, as I feel like this can be autogenerated. However, mind creating an issue? Right now, I am trying to get all the content in, and then will do some customization around it.